src/canvas-extractor/edfi_canvas_extractor/config.py

```diff
@@ -52,3 +55,21 @@ def get_canvas_api(canvas_base_url: str, canvas_access_token: str) -> Canvas:
         a new CanvasAPI object
     """
     return Canvas(canvas_base_url, canvas_access_token)
+
+
+MAX_TOTAL_CALLS = 4
+RETRY_WINDOW_AFTER_FIRST_CALL_IN_SECONDS = 600
```

Comment:
What the retry decorator does is to validate:
1. If the max total tries calls have not been reached.
1. Regardless of the number of tries, it validates if the total time of execution for the specific function has taken longer than this parameter, so sometimes if it does not have enough time, it won't retry to execute it.